### DISCUSSION TITLE
help with aircrack code

### DISCUSSION THREAD

**Discussion by TenEvil122** [aircrack-ng/aircrack-ng#2595] [General]
It's not actually a bug, because I don't know where I should report it, but I have this problem in this code and I needed help because I don't understand it very well, and I'm learning now.  (my phone is Android and I'm not rooted on it)
![Screenshot_20240111_083920_Termux](https://github.com/aircrack-ng/aircrack-ng/assets/156334426/9e36e136-a582-4b4e-8d43-b1a9e1b96617)

**Comment by Mister-X-** [comment]
I'm assuming you're running it on Android, and I'm guessing due to Android security, it doesn't work. 

If you're using Android, you'd have to use NetHunter.
